server/urls.js

```diff
@@ -40,6 +41,31 @@ function isImage(url){
 
 }
 
+function isVideo(url){
+
+	return new Promise(function(resolve, reject){
+		request(url, {timeout: RESPONSE_TIMEOUT})
+			.on('response', function(res){
+				res.on('end', () => reject('No data in response'));
+				res.destroy();
+			})
+			.on('data', function(chunk) {
```

Comment:
is it possible to close the stream early so that the whole file does not get downloaded.